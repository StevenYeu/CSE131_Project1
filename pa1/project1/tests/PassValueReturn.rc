function : int foo() {return 5.5;}
