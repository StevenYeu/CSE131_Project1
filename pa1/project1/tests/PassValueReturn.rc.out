Error, "/tmp/cs131_testrunner_OJ0qMF/file.rc": 
  Type of return expression (float), not assignment compatible with function's return type (int).
Compile: failure.
